source/components/recipe-contribute/recipe-contribute.js

```diff
@@ -1,11 +1,15 @@
-import { Database } from "../../core/database/database.js";
+import { Database } from "/core/database/database.js";
+import { Storage } from "/core/storage/storage.js";
 import { YummyRecipesComponent } from "/components/core/yummy-recipes-component.js";
 
 /** Class that provides functionality to the recipe contribute page. */
 class RecipeContribute extends YummyRecipesComponent {
   constructor() {
     super();
     this.htmlPath = "components/recipe-contribute/recipe-contribute.html";
+
+    // Flag to determine if image was input in the form
+    this["imageChanged"] = false;
```

Comment:
I think we should use instance properties, rather than array indices on `this`. See below:
```suggestion
    // Flag to determine if image was input in the form
    this.imageChanged = false;
```

Reply:
If you could replace all instances of `this["imageChanged"]` with `this.imageChanged`, that would be great.

Reply:
Will do